### ISSUE TITLE
Can you provide some information about train_loader.csv? And how I can recreate it for my custom dataset?

### ISSUE THREAD

**Issue by saumyaborwankar** [TaoRuijie/TalkNet_ASD#10]

**Comment by TaoRuijie** [comment]
For instance, the line in the loader csv file is:

HV0H6oc4Kvs_0960_1020:1	233	25.0	[0, 0, 0, 0, 0, 0, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 0, 0, 0, 0, 0, 0, 0, 0, 0, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 0, 0, 0, 0, 0, 0, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0]	2715

MP4 file id, length of clips, fps, ASD labels, index

In data loader, we load the label sequence based on these 'ASD labels', MP4 file id provide the name of the audio and visual file, so we can load the audio and visual file based on that.

**Comment by saumyaborwankar** [comment]
Thank you so much

**Comment by Falmi** [comment]
@TaoRuijie Thanks once again for the contribution. Could you tell us how you indexed each line in the CSV file, the last column?

**Comment by TaoRuijie** [comment]
Hi, please check here:

https://github.com/TaoRuijie/TalkNet-ASD/issues/30
